tiledb/sm/array_schema/domain.cc

```diff
@@ -1031,12 +939,16 @@ void Domain::compute_cell_num_per_tile() {
       compute_cell_num_per_tile<uint64_t>();
       break;
     default:
-      assert(0);
+      return;
   }
 }
 
 template <class T>
 void Domain::compute_cell_num_per_tile() {
+  // Applicable only to integer domains
+  if (!std::numeric_limits<T>::is_integer)
```

Comment:
Not sure if you want `std::is_integral<T>::value` instead here, maybe it doesn't matter.

Reply:
Yeah, either way should work.